Azure-README.md

```diff
@@ -54,7 +54,7 @@ Only Azure ARM templates are supported for deploying resources to Azure:
 `mytemplate.json` is located in resources. If you wish to share cloud resources among multiple test classes, set `perSuite` parameter to true, include such test classes in a JUnit5 suite and run the suite, not the test classes. Resources will be deleted once the suite is finished. If the parameter is set to false, resources are undeployed after the test class finishes.
 
 About `AzureArmTemplate` parameters:
-- `parameters` - define key-value parameters for Azure ARM template. String, integer, securestring and boolean types are supported. String values are converted into correct JSON structure for Azure ARM template.
+- `parameters` - define key-value parameters for Azure ARM template. String, integer, secure string and boolean types are supported. String values are converted into correct JSON structure for Azure ARM template.
```

Comment:
I'd prefer keeping it together due to https://azuresdkdocs.blob.core.windows.net/$web/java/azure-resourcemanager-datafactory/1.0.0-beta.1/com/azure/resourcemanager/datafactory/models/SecureString.html